unit_tests/test_rule_loader.cpp

```diff
@@ -0,0 +1,358 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+Copyright (C) 2023 The Falco Authors.
```

Comment:
```suggestion
Copyright (C) 2024 The Falco Authors.
```

Happy new year :partying_face: